dhis-2/dhis-api/src/test/java/org/hisp/dhis/schema/transformer/UserPropertyTransformerTest.java

```diff
@@ -59,7 +59,8 @@ public void testUserTransform()
         UserPropertyTransformer transformer = new UserPropertyTransformer();
         UserPropertyTransformer.UserDto userDto = (UserPropertyTransformer.UserDto) transformer.transform( user );
 
-        assertEquals( uuid.toString(), userDto.getId() );
+        // assertEquals( uuid.toString(), userDto.getId() );
```

Comment:
commented line

Reply:
Yes, we will enable this in 2.36 when users are only using uuid as identifiers.